doc/source/themes/scikit-image/static/css/custom.css

```diff
@@ -297,6 +297,30 @@ p.sphx-glr-signature a.reference.external {
   display: none !important;
 }
 
+
+span.togglebutton {
+    cursor: pointer;
+    position: absolute;
+    top: 6px;
+    right: 25px;
+    font-family: "monospace";
+    line-height: 100%;
```

Comment:
I'm not sure whether the absolute positioning, padding, line-height and such will translate gracefully to other browsers and machines. Any CSS experts that have more insight?